{"name": "Получение заказа пользователя без авторизации", "status": "passed", "start": 1709681637830, "stop": 1709681637986, "uuid": "97227048-703b-4c71-96a7-a436d7e8d341", "historyId": "4789c7aac39e50078c2e0c4aa7b44d12", "testCaseId": "4789c7aac39e50078c2e0c4aa7b44d12", "fullName": "tests.test_get_order.TestApiGetUserOrders#test_no_auth_user_get_user_order_error", "labels": [{"name": "feature", "value": "Проверяем получение заказа пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestApiGetUserOrders"}, {"name": "host", "value": "DESKTOP-AQ71LTK"}, {"name": "thread", "value": "18908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}